### PR TITLE
Convert to a ddit based build process.

### PR DESCRIPTION
**Pull request by mschaef-da**

### CODE REVIEW

**Review by da-tanabe — APPROVED** [review]

File: dabl-meta.yaml [hunk 1]
```diff
@@ -1,17 +1,11 @@
 catalog:
     name: dabl-chat
-    version: 0.1.5
+    version: 0.2.0
     short_description: DABL Chat
     description: A straightforward web chat app.
-    release_date: 2020-09-30
```
Comment (dimitri-da) [inline dabl-meta.yaml:6]: we no longer have a `release_date` field?

File: python/operator/setup.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
+import os
 from setuptools import setup
 
 setup(name='dablchat-operator-bot',
-      version='1.8.5',
+      version=os.environ['DDIT_VERSION'],
```
Comment (dimitri-da) [inline python/operator/setup.py:5]: that's nice, so we don't have to manually update the version everywhere

**Review by dimitri-da — APPROVED** [review]
looks good. Just one question regarding `release_date`

**Review by maxdeliso-da — APPROVED** [review]
```
     .  o ..                  
     o . o o.o                
          ...oo               
            __[]__            
         __|_o_o_o\__         
         \""""""""""/         
          \. ..  . /          
     ^^^^^^^^^^^^^^^^^^^^  
```

File: python/operator/setup.py (hunk 2 above)
Comment (mschaef-da) [reply python/operator/setup.py:5]: Yeah... it also keeps the versions in sync across sub-artifacts, which I think makes more sense given that they're all packaged in the same file.

File: dabl-meta.yaml (hunk 1 above)
Comment (mschaef-da) [reply dabl-meta.yaml:6]: It's still there, but now automatically populated by the build tool.  `ddit` will parse `dabl-meta.yaml`, make a few updates and checks of this sort, and then rewrites the result into the DIT file. (This is also why the file no longer explicitly contains subdeployment listings. You _can_ have them, but the tool looks at the command line arguments and DAR file build and includes those automatically.)

**Pull request merged**